diagnostic.md

```diff
@@ -140,7 +166,7 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+5,4,3,2,1 from left to right on one line
```

Comment:
Oh yeah, I was wondering about this. Isn't div1 going to float all of the subsequent divs right with it? And since nothing clears right or clears both they'll all follow along?